src/shogun/lib/observers/ParameterObserverLogger.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * This software is distributed under BSD 3-clause license (see LICENSE file).
+ *
+ * Authors: Giovanni De Toni
+ *
+ */
+#ifndef SHOGUN_PARAMETEROBSERVERLOGGER_H
+#define SHOGUN_PARAMETEROBSERVERLOGGER_H
+
+#include <shogun/lib/observers/ParameterObserver.h>
+
+namespace shogun {
+
+	/**
+	 * This class implements a logger which prints all observed updates.
+	 */
+	class ParameterObserverLogger : public ParameterObserver {
```

Comment:
I would go for "ParameterLogger" but that is taste I guess. You will have noticed that I like short names these days. Really inspired by Sergey on that :)

Reply:
mmh I would be in favour of keeping the "Observer" somewhere, so it becomes clear about what is this object :)